components/bl-transitions-component/README.md

```diff
@@ -52,3 +52,15 @@ For example:
 <img alt="Close button logic" src="./example-images/close_button_logic.jpg" width="900" />
 
 In the preview, you now have the option to change the visibility of the component with animation by clicking on either the Close or Open button.
+
+### Adding component with dynamic content in transition
+It can come in handy when animating a block that requires some time to render completely. To good utilize the Collapse Top and Collapse Left effects, it's important to employ this guid.
+1. Firstly, you need activate it in the settings:
+
+> WARNING: After this action the component will not be displayed
+
+<img alt="Close button logic" src="example-images/enable_dynamic_content.jpg" />
+
+2. Secondly, your block must have event that triggered when he full loaded. In him, we add component action "Set Content Loaded"
```

Comment:
Remove first and secondly words, you already have ordered list with numbers

Reply:
Also, what about rename actions `Set Content Loaded Transitions: <transition name>`  for `Set Content Loaded for: <transition name>`

Ask @snaqqqystas what he think about it, and if we decide to do it, please remake screenshots.

Reply:
it's ok, but with it maybe change the effects raw and default effect to another one, - and if user will use the image without props, default effect normally work?

Reply:
i've made another ticket for default effect changing.